tests/business-logic/Users.test.ts

```diff
@@ -0,0 +1,84 @@
+import UserEntity from "@business-logic/User";
+
+import { hashPassword } from "@helpers/auth";
+import prisma from "@helpers/prisma";
+import { teardown } from "@helpers/tests/teardown";
+
+let hashedPassword:string;
+let userEmail = "lambert@gmail.com";
```

Comment:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/98182227/160559772-becd469d-8a41-452e-a83d-a509dc979904.png">

Reply:
the PR doesn't seem to be ready for review yet there are still some pipeline errors. can you please always resolve the pipeline errors before submitting a PR?

<img width="1470" alt="image" src="https://user-images.githubusercontent.com/98182227/160559999-6b6e2c78-b2c0-4f5d-8345-2fe7f54c06c7.png">